README.md

```diff
@@ -9,12 +9,15 @@
 
 ## Introduction
 
-This library can handle IPv4, IPv6 addresses, as well as IP ranges, in CIDR formats (like `::1/128` or `127.0.0.1/32`) and in pattern format (like `::*:*` or `127.0.*.*`).
```

Comment:
I've picked some of these great fixes of README.md: see https://github.com/mlocati/ip-lib/commit/790e2311720ef1f119f40233215e121b583e12e6